README.md

```diff
@@ -0,0 +1,7 @@
+Workshop Czechitas
```

Comment:
Dal bych pryc, je to tu zbytecny, tohle bude v pokynech (mimo to je to anglicky)

Reply:
taky je to readme .. tohle jim posilat nebudem

Reply:
No prave, tak proc to tu mit, navic v tech krocich. To muzeme pridat na konec do masteru.